scapy/layers/inet.py

```diff
@@ -387,14 +387,6 @@ def extract_padding(self, s):
         l = self.len - (self.ihl << 2)
         return s[:l],s[l:]
 
-    def send(self, s, slp=0):
```

Comment:
I don't think it should, if my memory serves well it used by `[iface].send()` to select the interface (and by `SourceMACField`, I've just checked).